README.md

```diff
@@ -44,11 +44,33 @@ We heartily welcome external contributions to Marathon's documentation. Document
 
 ## Setting Up And Running Marathon
 
+### Dependencies
+Marathon has the following compile-time dependencies:
+* sbt - A build tool for scala. You can find the instructions for installing sbt for Mac OS X and Linux over [here](http://www.scala-sbt.org/0.13/tutorial/Setup.html).
+* JDK 1.8+
+
+For run-time, Marathon has the following dependencies:
+* libmesos - JNI bindings for talking to Apache mesos master. Look at the *Install Mesos* section for instructions to get libmesos.
+* Apache Zookeeper - We can have a spearate Zookeeper installation specifically for Marathon, or we can use the same Zookeeper used by Mesos.
+
 ### Installation
 
 #### Install Mesos
+Marathon requires libmesos, a shared object library, that contains JNI bindings for Marathon to talk to the mesos master. *libmesos* comes as part of the Apache mesos installation. There are two options for installing Apache mesos. 
+
+##### Installing Mesos from prepackaged releases 
+Instructions on how to install prepackaged releases of Mesos are available [in the Marathon docs](https://mesosphere.github.io/marathon/docs/).
+
+##### Building  Mesos from source
+**NOTE:** *Choose this option only if building Marathon from source, else there might be version incompatability between pre-packaged releases of Marathon and Mesos built from source.*
```

Comment:
This was aimed at newbie mesos developers (like me) who might find this information useful. I agree it is available at other sources as well, but thought since the location of libmesos.so/dylib is critical to marathon might be explicit over here.